docs/docs/deploy/heroku.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Deploy to Heroku
+summary: A walkthrough of deploying to heroku
+authors:
+    - Stephen van Beek
+date: 2020-05-27
+---
+
+# Deploying to Heroku
+
+[Heroku](https://www.heroku.com) is a PaaS product that makes deploying your application easy. With it's generous free tier it's a perfect way to kick off a new project.
+
+Springboard works pretty well with Heroku and benefits a fair bit from a number of Heroku add-ons:
```

Comment:
This is wishy washy. "works pretty well" "a fair bit". 

"It's easy to deploy a Springboard application with Heroku, with the benefit of a number of Heroku add-ons to enhance the application easily." 

Or something like this